scripts/teardown.sh

```
@@ -0,0 +1,172 @@
+echo "Deleting AAP operator..."
+oc delete -f prerequisites/aap-installation.yaml --ignore-not-found --wait=false 2>/dev/null || true
+oc delete namespace ansible-aap --ignore-not-found --wait=false 2>/dev/null || true
+
+# Optionally delete Multicluster Engine (before LVMS since AgentServiceConfig PVCs depend on storage)
+if [[ "${MCE_SERVICE}" == "true" ]]; then
+    echo "Deleting AgentServiceConfig..."
+    oc delete agentserviceconfig agent --ignore-not-found --timeout=120s 2>/dev/null || true
+    echo "Deleting MultiClusterEngine..."
+    oc delete multiclusterengine --all --ignore-not-found --timeout=120s 2>/dev/null || true
+    # Wait for MultiClusterEngine to be fully removed
+    retry_until 120 5 '[[ -z "$(oc get multiclusterengine --no-headers 2>/dev/null)" ]]' || {
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)